src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -402,6 +405,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
                 { ShortcutAction::Quit, RS_(L"QuitCommandKey") },
                 { ShortcutAction::AdjustOpacity, L"" }, // Intentionally omitted, must be generated by GenerateName
                 { ShortcutAction::RestoreLastClosed, RS_(L"RestoreLastClosedCommandKey") },
+                { ShortcutAction::SelectCommand, MustGenerate },
+                { ShortcutAction::SelectOutput, MustGenerate },
```

Comment:
I mean, I'm down to use `MustGenerate`, but you might as well replace the other `L""` while you're here, right?